server/src/main/java/org/apache/iotdb/db/conf/IoTDBConfig.java

```diff
@@ -1309,6 +1311,21 @@ public boolean isEnablePerformanceTracing() {
 
   public void setEnablePerformanceTracing(boolean enablePerformanceTracing) {
     this.enablePerformanceTracing = enablePerformanceTracing;
+    if (!enablePerformanceTracing) {
+      try {
+        TracingManager.getInstance().close();
```

Comment:
generate a class only do a thing.  IoTDBConfig only get or set config.. if you need deal other logic, you can do it in other class